src/EntityGraphQL/Schema/SchemaProvider.cs

```diff
@@ -45,6 +45,8 @@ public class SchemaProvider<TContextType> : ISchemaProvider
 
         public IDictionary<Type, ICustomTypeConverter> TypeConverters { get; } = new Dictionary<Type, ICustomTypeConverter>();
 
+        public List<WhitelistedException> WhitelistedException { get; } = new();
```

Comment:
As we use `IsAllowed` below can we name this `AllowedExceptions`?

Reply:
Yep, done!